mne/viz/evoked.py

```diff
@@ -1355,6 +1352,32 @@ def _ci(arr, ci):
                        for mean_, sigma_ in zip(mean, sigma)]).T
 
 
+def _bootstrap_ci(arr, ci=.95, n_bootstraps=2000, statfun='mean'):
+    """Get confidence intervals from non-parametric bootstrap."""
+    if statfun == "mean":
+        statfun = lambda x: x.mean(axis=0)  # noqa
+    elif statfun == 'median':
+        statfun = lambda x: np.median(x, axis=0)  # noqa
+    elif not callable(statfun):
+        raise ValueError("statfun must be 'mean', 'median' or callable.")
+    ntrials, ntimes = arr.shape
+    indices = np.arange(ntrials, dtype=int)  # BCA would be cool to have too
+    boot_indices = np.random.choice(indices, replace=True,
+                                    size=(ntrials, len(indices)))
```

Comment:
not allowing to pass a random_state is dangerous. Please pass one to the function and use check_random_state function